### PR TITLE
Remove ci-pipeline functions from this repo

### PR DESCRIPTION
**Pull request by johnbieren**
Remove pipelineUtils functions from this repo so this repo is not dependent on the shared library located in ci-pipeline. 

I still have to finish up ContainerBuildTrigger, ContainerTaskTrigger, and ContainerJenkinsfile before merging, but I figured I'd open the PR now so you can start reviewing the standard workflow parts.

This will make rebasing the updated message spec PR a pain, but I probably have a bit of time to do that since we are waiting on resultsDB to be updated =)

### CODE REVIEW

File: JenkinsfileTaskTrigger [hunk 1]
```diff
@@ -65,44 +65,47 @@ timestamps {
...
-                        if (env.fed_info_request_0) {
+                        parsedMsg = kojiMessage(message: env.CI_MESSAGE, ignoreErrors: true)
+                        if (parsedMsg.has('info')) {
                             env.isScratch = true
+                            env.request_0 = parsedMsg['info']['request'][0]
+                            env.request_1 = parsedMsg['info']['request'][1]
+                        } else {
+                            env.isScratch = false
+                            env.request_0 = parsedMsg['request'][0]
+                            env.request_1 = parsedMsg['request'][1]
```
Comment (bgoncalv) [inline JenkinsfileTaskTrigger]: this code to set env.request and isScratch was used in a couple of files, wouldn't be better to have a function for it?

**Comment by bgoncalv** [comment]
It looks good, but this is not urgent change. 
Would be easier to wait for it until the updated message spec PR is merged?

File: JenkinsfileTaskTrigger (hunk 1 above)
Comment (johnbieren) [reply JenkinsfileTaskTrigger]: Good suggestion, I can do that

**Comment by johnbieren** [comment]
Sure, that's fine. I'll leave this on the back burner until we get the message spec PR merged. Also btw I added that retry to checkTests in contra-lib, not ci-pipeline, so I don't know how badly we needed that change, but the new code won't be called until this PR is merged

**Comment by bgoncalv** [comment]
I don't think the fix for checkTests is that urgent either. I believe we can wait for this PR to be merged.

**Comment by bgoncalv** [comment]
@johnbieren I think this can be merged without the new format of message PR. :)

**Comment by johnbieren** [comment]
@bgoncalv do you mean we no longer have to wait for the PR we discussed above? So I should rebase and we can see if it performs well in staging then merge if it does?

**Comment by bgoncalv** [comment]
@johnbieren we can confirm with @thrix first, but I think this effort to change the message format is kind on hold for now...

**Comment by thrix** [comment]
Yeah, due to lack of time, the change of messaging is on hold for now ... As this is not trying to change the messaging, I believe we can move with this forward ...

**Comment by johnbieren** [comment]
[test]

**Comment by bgoncalv** [comment]
[merge]

**Pull request merged**